mongo/testgcpkms/main.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) MongoDB, Inc. 2022-present.
```

Comment:
Optional: Executable targets are typically in `cmd/<some package>/main.go`. Consider moving this file to `cmd/testgcpkms/main.go`.

I've created [GODRIVER-2517](https://jira.mongodb.org/browse/GODRIVER-2517) that describes moving all of the `main.go` files that are binary build targets into the `cmd/` directory.

Reply:
Done. And thank you for creating GODRIVER-2517.